app/src/main/res/values/strings.xml

```diff
@@ -141,6 +141,23 @@
     <string name="end_time_is_less_than_start_time">End time is less than start time</string>
     <string name="you_have_to_choose_the_from_time_first">  You have to choose the from time first</string>
 
+    <!-- Validate Request Leave -->
+    <string name="the_working_time_dose_not_fit_to_the_request">The working time dose not fit to the request</string>
+    <string name="form_overdue">Form overdue</string>
```

Comment:
FATAL: [MissingTranslation] "`form_overdue`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)